nativescript-3.1.0-timeline/README.md

```diff
@@ -0,0 +1,236 @@
+# NativeScript 3.1.0 Timeline Introduction
+Performance has been a high priority feature in {N} all along its development. During the course of development we found ourselves manually instrumenting all of the Objective-C, Java, C, C++ and the JavaScript code several times, but so far these profile traces were kept for internal use only and never made it to the "master".
+
+With 3.1.0 we have "merged" the manual instrumentation and enabled all {N} apps to trace execution times of some key components. Here is how to consume this.
+
+## Enable Traces
+In the `app/package.json` set `profiling` to `timeline`:
+```json
+{
+    "main": "MyApp",
+    "profiling": "timeline"
+}
+```
+This will enable the traces and will output times in the console directly.
+
+The feature is in experimental state and we are tunning it with the @next releases, so if you want to use it consider installing the next versions of the `tns-core-modules` and the run-times.
+```bash
+tns platform add android@next
+npm i tns-core-modules@next
+```
+
+Running the app now will be pretty verbose.
+
+```bash
+CONSOLE LOG Timeline: Runtime: Expose: UIViewController  (98155132.912ms - 98155218.291ms)
+CONSOLE LOG Timeline: Runtime: Expose: UIView  (98155124.914ms - 98155221.567ms)
+CONSOLE LOG Timeline: Runtime: Expose: UIResponder  (98155124.841ms - 98155234.224ms)
+CONSOLE LOG Timeline: Runtime: Expose: UIScreen  (98155124.591ms - 98155238.539ms)
+CONSOLE LOG Timeline: Runtime: require: /app/tns_modules/tns-core-modules/utils/utils.js  (98155114.122ms - 98155239.010ms)
+CONSOLE LOG Timeline: Runtime: require: /app/tns_modules/tns-core-modules/image-source/image-source.js  (98155239.113ms - 98155260.134ms)
+CONSOLE LOG Timeline: Runtime: require: /app/tns_modules/tns-core-modules/ui/styling/background.js  (98155097.347ms - 98155268.315ms)
+CONSOLE LOG Timeline: Runtime: require: /app/tns_modules/tns-core-modules/ui/core/properties/properties.js  (98155295.330ms - 98155313.461ms)
+```
+
+The traces times are the start and end time of the event in milliseconds since 1970.
+
+At one hand we can now use these traces to ensure certain points in the execution of the app are reached in time. On the other you get an overview of the processes that takes place when your app executes.
```

Comment:
At -> `On` one hand...

Can we reduce the two sentences into something like: `You now get an overview of the processes that take place during your app's execution, and how long they take.`